pkg/scannerv4/mappers/mappers_test.go

```diff
@@ -899,6 +899,223 @@ func Test_toProtoV4Contents(t *testing.T) {
 	}
 }
 
+func Test_toProtoV4VulnerabilitiesMapWithEPSS(t *testing.T) {
+	now := time.Now()
+	protoNow, err := protocompat.ConvertTimeToTimestampOrError(now)
+	assert.NoError(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```

Reply:
Not sure why, `require` gives me error in the file

Reply:
fine to just do assert since it's used in other places in this file, but I imagine if you are just committing the suggestion as-is, it's because `"github.com/stretchr/testify/require"` is not imported

Reply:
I think it's not find to use `assert` because it will not fail the test and will continue execution, am I wrong?

Reply:
In general, I agree. I tend to do `require` for anything that I consider "setup" for the test, which this definitely is. I figured since another test in this file uses `assert` for the same thing, there is already that precedence. That plus the urgency of getting this in, I am willing to let this slide, but I'm also all for switching it to `require`, time permitting

Reply:
Merging this PR for nightly build, I will address this concern in another PR